templates/etc/haproxy/haproxy-frontend.cfg.j2

```diff
@@ -44,6 +44,9 @@ frontend {{ name }}
     acl             {{ acl }}
                 {% endfor %}
             {% endif %}
+            {% if value.monitor is defined %}
```

Comment:
I guess you mean `value.monitor_fail` ?

Reply:
Of course, thanks for catching it, that's what I get for writing it manually instead of copying from testing...